newsletter_automation/newsletter/routes.py

```diff
@@ -21,6 +22,8 @@
 from prometheus_client import generate_latest, CONTENT_TYPE_LATEST
 from prometheus_client import Counter, Gauge
 import psutil
+import time
+from functools import wraps
```

Comment:
Where are we using wraps?

Reply:
I had added it as a part of the first implementation approach, forgot to remove the import. Removed now